dotCMS/src/main/java/com/dotmarketing/util/ConvertToPOJOUtil.java

```diff
@@ -223,215 +140,131 @@ private static Link getLinkFields(Map<String, String> map) throws ParseException
      * @param results
      * @return
      */
-    public static List<Identifier> convertDotConnectMapToIdentifier(final List<Map<String,String>> results)
+    public static List<Identifier> convertDotConnectMapToIdentifier(final List<Map<String, Object>> results)
             throws ParseException {
         final List<Identifier> ret = new ArrayList<>();
-        if(results == null || results.isEmpty()){
+        if (results == null || results.isEmpty()) {
             return ret;
         }
 
-        for (Map<String, String> map : results) {
+        for (Map<String, Object> map : results) {
             final Identifier i = new Identifier();
-            i.setAssetName(map.get("asset_name"));
-            i.setAssetType(map.get("asset_type"));
-            i.setHostId(map.get("host_inode"));
-            i.setId(map.get("id"));
-            i.setParentPath(map.get("parent_path"));
-
-            if (map.get(SYSPUBLISH_DATE) != null && !map.get(SYSPUBLISH_DATE).isEmpty()){
-                i.setSysPublishDate(df.parse(map.get(SYSPUBLISH_DATE) ));
-            }
-
-            if (map.get(SYSEXPIRE_DATE) != null && !map.get(SYSEXPIRE_DATE).isEmpty()){
-                i.setSysExpireDate(df.parse(map.get(SYSEXPIRE_DATE) ));
-            }
-
+            i.setAssetName((String) map.get("asset_name"));
+            i.setAssetType((String) map.get("asset_type"));
+            i.setHostId((String) map.get("host_inode"));
+            i.setId((String) map.get("id"));
+            i.setParentPath((String) map.get("parent_path"));
+            i.setSysPublishDate((Date) map.get(SYSPUBLISH_DATE));
+            i.setSysExpireDate((Date) map.get(SYSEXPIRE_DATE));
             ret.add(i);
         }
         return ret;
     }
 
-    public static List<Folder> convertDotConnectMapToFolder(final List<Map<String,String>> results)
-            throws ParseException {
+    public static List<Folder> convertDotConnectMapToFolder(final List<Map<String, Object>> results) throws ParseException {
 
         List<Folder> ret = new ArrayList<>();
 
-        if(results != null && !results.isEmpty()){
-            for (Map<String, String> map : results) {
+        if (results != null && !results.isEmpty()) {
+            for (Map<String, Object> map : results) {
                 ret.add(getFolderFields(map));
             }
         }
         return ret;
     }
 
     @NotNull
-    private static Folder getFolderFields(Map<String, String> map) throws ParseException {
+    private static Folder getFolderFields(Map<String, Object> map) throws ParseException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.text.ParseException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)